model/scikit/dffml_model_scikit/scikit_base.py

```diff
@@ -240,29 +241,31 @@ class Scikit(Model):
     def __init__(self, config) -> None:
         super().__init__(config)
         self.saved = {}
+        self.hashlib = importlib.import_module("hashlib")
+        self.json = importlib.import_module("json")
```

Comment:
Same with `json`, regular `import` statement for modules that are in the standard library